apps/admin/src/pages/sponsors/update-sponsor-form.tsx

```diff
@@ -69,6 +69,9 @@ export function UpdateSponsorForm({ sponsor }: { sponsor: Sponsor }) {
   // fill array with all valid hackathonIds
   const { data: hackathons } = trpc.hackathons.getAll.useQuery();
 
+  // Initialize tiers array
+  const tiers = ["gold", "silver", "bronze"];
```

Comment:
See my first comment.

Reply:
That was actually your second comment big bro